md/rest_extensions.md

```diff
@@ -4,10 +4,13 @@
 
 API used by Operations Management Living Application
 
-
 ## `/bici-case-api`
-### `GET`
+#### Http method
+`GET`
+
+#### Description
 Get list of running cases
```

Comment:
"Get the list of open cases."
"open cases" is the term used in the case list. It's worth reusing everywhere.